src/contracts/increaseStake.ts

```diff
@@ -0,0 +1,92 @@
+import IUniswapV2ERC20 from "@uniswap/v2-core/build/IUniswapV2ERC20.json";
+import { ethers } from "ethers";
+import { parseUnits, randomBytes } from "ethers/lib/utils";
+import { signPermission, signPermitEIP2612 } from "./utils";
+import aludelAbi from "./aludelAbi";
+import Crucible from "./Crucible.json";
+import transmuterAbi from "./transmuterAbi";
+
+export async function increaseStake(
+  signer: any,
+  crucibleAddress: string,
+  rawAmount: string
+) {
+  const walletAddress = await signer.getAddress();
+
+  // Recipient doesn't make much sense when increasing stake
+  const args = {
+    crucible: crucibleAddress,
+    aludel: "0xf0D415189949d913264A454F57f4279ad66cB24d",
+    transmuter: "0xB772ce9f14FC7C7db0D4525aDb9349FBD7ce456a",
+    crucibleFactory: "0x54e0395CFB4f39beF66DBCd5bD93Cca4E9273D56",
+    recipient: walletAddress,
```

Comment:
Looks like recipient is not being used, should it be removed?

Reply:
thanks